katas/squaredigit/joaquin/SquareDigit.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace SquareDigit.Joaquin {
+    public class SquareDigitClass {
+        public int squareDigits(int input) {
+            string numbers = input.ToString();
+            string result = "";
+
+            foreach (char c in numbers) {
+                double squared = Math.Pow(Int32.Parse(c.ToString()),2);
```

Comment:
Don't forget to add one space after each comma, helps readabillity and good coding style